ui/apps/platform/src/types/centralCapabilities.ts

```diff
@@ -0,0 +1,8 @@
+// Semicolon on separate line following the strings prevents an extra changed line to add a string at the end.
+// However, add strings in alphabetical order to minimize merge conflicts when multiple people add strings.
+// prettier-ignore
+export type CentralCapabilitiesFlags =
```

Comment:
Can you see if it MetadataService.ts can export the equivalent string enumeration as follows?

```ts
export type CentralCapabilitiesFlags = keyof CentralServicesCapabilities;
```

https://www.typescriptlang.org/docs/handbook/2/keyof-types.html#the-keyof-type-operator

Reply:
Much better^